pynance/textimporter_test.py

```diff
@@ -244,6 +250,147 @@ def test_csv_importer_read_dkbvisa_text(self):
                     "REWE Markt GmbH ZW"]
         self.assertListEqual(expected, list(result_df["text"].values))
 
+    def test_dkbvisa_StringIO_import(self):
```

Comment:
I don't know if this would be pythonic, but wouldn't it make sense to extract all the total balance tests into a separate subclass of `unittest.TestCase`? That might make the naming of test methods easier as well...

Reply:
yes you're right. I found it became impossible to navigate through the tests. Will fix